### DISCUSSION TITLE
Processing a Weblate translation point

### DISCUSSION THREAD

**Discussion by jean-louis67** [photoprism/photoprism#2467] [Q&A]
Hello,

I have an alert message from weblate: _The source string and the translated string are identical._
In the particular case shown below, this is normal.
How can I indicate that there is no error, so validate the translation, and thus make this alert disappear?
I can click Ignore or Reset, check Verify, etc... it doesn't seem to have any effect.
Thank you for your help
![image](https://user-images.githubusercontent.com/65403912/175803419-edaf6fb9-ae87-4010-9a51-ef0e1f9a956f.png)

**Comment by graciousgrey** [comment]
I don't see this check when filtering for "any failing checks" so I guess you ignored it successfully :)

It looks like you are filtering for ignored checks ("Chaînes traduites avec des verifications de qualité ignorées")
